Kop3 Onboarding Project#1

KacperParaniuk opened the pull request:
FInished the first project of creating a simple circuit schematic and pcb layout of a led circuit.

KacperParaniuk committed:
First Commit - File setup

KacperParaniuk committed:
Project #1 Complete

Created a simple led circuit using KiCad software. Created a schematic and a subsequent pcb layout.

KacperParaniuk committed:
Onboarding Project - Firmware Project #1 Heartbeat

Initialized a new led to pin 4 on the MCU, added code to check for when there is a 1 second interval to change the led from on to off or vise versa.

KacperParaniuk committed:
Specs File - Horns Breakout Board Project

Described the purpose, function and necessities of the Horns Breakout Board.

KacperParaniuk committed:
Horn Breakout Board Schematic

Created the Horn Breakout Board Schematic following the wiki steps. This includes decoupling capacitors, half of an rc filter, and two horns in series.

KacperParaniuk committed:
PCB Layout for Horn Breakout Board Done.

Created the PCB for the Horn Breakout Board.

KacperParaniuk committed:
Project 2: Potentiometer Blink Rate

Finished Project 2 of Firmware and fixed an issue from project 1 of not setting a particular LED high/low

johnathanvogt reviewed (commented):
Firmware looks good. Please update connector footprints to
Connector_Molex:Molex_Micro-Fit_3.0_43650-0300_1x03_P3.00mm_Horizontal
Connector_Molex:Molex_Micro-Fit_3.0_43650-0300_1x02_P3.00mm_Horizontal
Otherwise, basic firmware looks good. You will need to remove the push button switch in  your horns board. here, the MOSFET acts as a switch that activates the horn modules, so we will not need a push button. Please redirect the diode to allow some current to flow back through the horn modules and stop a direct connection between 24v and ground.

KacperParaniuk committed:
Onboarding Check FIX

Switched the connector footprints, and rearranged the rectifier diode to protect against current back flow and from high current flowing through the mosfet/tripping the fuse. Re-arranged the PCB layout.
